### PR TITLE
Related articles show up in 3 view tab and refactored news API call to util file

### PR DESCRIPTION
**Pull request by htn5**
Update to the 3 view tab on the "Article Synopsis Screen" to show the Related Articles based off of the hotspotID. A second API call is made to generate these articles, but to be changed so that it calls the articles from the database store instead. (hotpots generated by top articles and stored in DB > articles categorized > related articles tab shows articles stored under that category)

### CODE REVIEW

File: GeoNewsFinder4/components/BottomSheet.js [hunk 1]
```diff
@@ -2,45 +2,24 @@ import { React, useEffect, useState } from 'react';
...
-      result = await result.json();
-      setData(result.articles);
-    }
     useEffect(() => {
-      if(hotspotId) {
-        getAPIdata();
-      }
-    }, [hotspotId]);
+      const fetchData = async () => {
+        console.log("fetching data")
```
Comment (mattn02) [inline GeoNewsFinder4/components/BottomSheet.js]: Can we remove this console.log to remove clutter

File: GeoNewsFinder4/components/RelatedArticles.js [hunk 2]
```diff
@@ -1,17 +1,67 @@
...
+import { View, Text, TouchableOpacity, StyleSheet, FlatList, Image } from 'react-native';
+import { Card } from 'react-native-elements';
+import getAPIdata from '../utils/getAPIdata';
 
-const RelatedArticlesRoute = () => {
+const RelatedArticlesRoute = ({ route }) => {
+
+  const [newsData, setData] = useState([]);
+  useEffect(() => {
+    const fetchData = async () => {
```
Comment (mattn02) [inline GeoNewsFinder4/components/RelatedArticles.js:10]: might be more efficient to declare this fetchData outside of the use effect so we don't have to "generate" a function variable fetchData every time useEffect gets called.

**Review by mattn02 — CHANGES_REQUESTED** [review]
fix changes but lgtm

**Review by K3nnneth — APPROVED** [review]
LGTM

nit: remove unnecessary console logs

**Review by mattn02 — APPROVED** [review]
lgtm awesome

**Pull request merged**